src/mesh/config/MeshConfiguration.cpp

```diff
@@ -113,7 +113,7 @@ void MeshConfiguration::addMesh(
         break;
       }
     }
-    PRECICE_ASSERT(found, "Data " << dataNewMesh->getName() << " is not defined. Please define a data tag with name=\"" << dataNewMesh->getName() << "\".");
+    PRECICE_CHECK(found, "Data {0} is not defined. Please define a data tag with name=\"{0}\".", dataNewMesh->getName());
```

Comment:
`PRECICE_CHECK` now wraps `fmt::format`. Do I understand this correctly? So you don't need to write any `fmt::format` here.